### ISSUE TITLE
Feature Request: Conversations

### ISSUE THREAD

**Issue by trymeouteh** [jhaals/yopass#2538]
To be able to create an E2EE conversation.

No accounts required.

When you encrypt a message, there will be an option to make it a conversation. When this setting is checked, a name field will be added which is optional to be filled out. When you view the encrypted message, it will show the name if it was filled out, message and a reply form which has an optional name field and message field.

The conversation will expire after X amount of time since the last message was sent. If the last message was sent 7 days ago and the conversation expiry was set to 7 days, the conversation will expire.

The conversation will only keep so many previous messages in the conversation. The host of the server can set the length of how many previous messages are kept in a conversation.

This will allow people who do not have private email account such as a ProtonMail or Tutanota to create a private email conversation.

**Comment by eponerine** [comment]
Agreed. Just use any other E2EE tool that exists. If you want it self-hosted, here are some options:

https://github.com/RishiBakshii/mern-chat
https://github.com/muke1908/chat-e2ee

**Comment by jhaals** [comment]
Thanks for the suggestion but this is not in scope for this software, I'm sure other solutions can do the job you're asking.
